DTS-STN/Life-Journeys
# Lj 268 connect to local resources

# Description

[Connect to local resources](https://lj-decd.atlassian.net/browse/LJ-268?atlOrigin=eyJpIjoiMTE1YTAxYjhiYWQ4NGNhOTliM2UxNWFjMzA0ZTI0MDYiLCJwIjoiaiJ9)

I added the `ProvincialLink` component which changes depending on the user's region. I also modified the `MoreInfo` links to be in line with the current Figma designs. At this point in time I deemed it better to have these as separate components as they do different things but down the road we can look at consolidating them using something like a ternary condition. I also made both the `ProvincialLink` and `MoreInfo` components open in a new tab. I've kept the links inside the first accordion for the time being but this will obviously change.

Note: It looks like a couple provincial sites don't have French translations (ex. BC and Alberta) so that link is the same regardless of the current locale.

Edit: Changed the array of objects so they included multiple links for each region (and Canada), mostly based on an excel spreadsheet provided by Steph but some links (mainly the child care ones) were the result of my own research so those are still subject to change. I updated Snyk to 1.634.0 as per the current open PR. I also altered `forms.css` so the checkbox for the Report a Problem component would be functional again.

## Test Instructions

1. Pull in branch
2. Type `npm run dev`
3. When prompted if you want to share your location, select `Block`. This will ensure that the region is set to `Canada` to begin with, so you can see that the provincial link will link to Canada.ca
4. Open up the first accordion 
5. Cycle through the provinces (and also switch between EN and FR) using the dropdown list and observe the link changing to the correct provincial website.

## Meets Definition of Done
